src/popup/router/components/TransactionItem.vue

```diff
@@ -1,36 +1,55 @@
 <template>
   <div class="transaction-item">
     <div>
-      <div class="status"><TokenAmount data-cy="amount" :amount="txAmount" /> {{ status }}</div>
+      <div class="status">
+        <TokenAmount
+          data-cy="amount"
+          :amount="txAmount"
+          :symbol="contractCallData ? availableTokens[contractCallData.token].symbol : 'AE'"
+        />
+        <span :class="status.value">{{ status.text }}</span>
+      </div>
       <span data-cy="time">{{ transaction.microTime | formatDate }}</span>
     </div>
     <div class="details">
-      <button v-if="tipUrl" class="url" @click="openUrl(tipUrl, true)">{{ tipUrl }}</button>
-      <span v-else-if="topup || withdraw" class="address">
+      <button
+        v-if="tipUrl || (contractCallData && contractCallData.url)"
+        class="url"
+        @click="openUrl(tipUrl || contractCallData.url, true)"
+      >
+        {{ tipUrl || contractCallData.url }}
```

Comment:
Proposing to add `contractCallData.url` below to how `tipUrl` is calculated, to don't overcomplicate the markup